compiler/crates/relay-typegen/src/config.rs

```diff
@@ -9,9 +9,26 @@ use fnv::FnvHashMap;
 use interner::StringKey;
 use serde::Deserialize;
 
+#[derive(Debug, Deserialize)]
+#[serde(deny_unknown_fields, rename_all = "camelCase")]
+pub enum TypegenLanguage {
+    Flow,
+    Typescript,
+}
+
+impl Default for TypegenLanguage {
+    fn default() -> Self {
+        Self::Flow
```

Comment:
I was even thinking of making Typescript the default, but just realized `*.ts` files wouldn't work with vanilla JS anymore. The Flow generated `*.js` files using Flow comment style syntax allows people to just treat them as pure JS files.

Maybe we could even make this optional and skip generating types altogether to make the files a bit smaller.